src/DependencyInjection/Compiler/ExtensionCompilerPass.php

```diff
@@ -96,13 +97,14 @@ public function process(ContainerBuilder $container)
     protected function getExtensionsForAdmin($id, Definition $admin, ContainerBuilder $container, array $extensionMap)
     {
         $extensions = [];
-        $classReflection = $subjectReflection = null;
 
         $excludes = $extensionMap['excludes'];
         unset($extensionMap['excludes']);
 
         foreach ($extensionMap as $type => $subjects) {
             foreach ($subjects as $subject => $extensionList) {
+                $class = null;
+
                 if ('admins' === $type) {
```

Comment:
```suggestion
                if ('admins' === $type && $id === $subject) {
                    $extensions = array_merge($extensions, $extensionList);
                    
                    continue;
                }
                
                $class = $this->getManagedClass($admin, $container);
                if (!class_exists($class)) {
                    continue;
                }
```

Reply:
But it could be even better if `isExtensionClass` call `getManagedClass` to get the class.
This way you could write 
```
                if ('admins' === $type && $id === $subject || $this->isExtensionClass($type, $subject)) {
                    $extensions = array_merge($extensions, $extensionList);
                }
```

Reply:
> But it could be even better if `isExtensionClass` call `getManagedClass` to get the class.

I plan to reuse the `isExtensionClass` with the next PR.